과제2/style2.css

```
@@ -0,0 +1,201 @@
+* {
+  margin: 0;
+  padding: 0;
+  text-align: center;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+  grid-template-rows: repeat(22, 100px);
```

```suggestion
  grid-template-rows: repeat(22, 100%);
```
절댓값으로 계산을 하면 screen 마다 여백의 차이가 있습니다. (가장 넓을 때와 중간 크기일 때 최하단에 여백이 남습니다)
계산이 복잡할 땐 100%로 설정하면 화면 세로길이에 꽉 채워집니다